draft-ietf-httpbis-connect-tcp.md

```diff
@@ -44,9 +44,9 @@ HTTP/3 uses a UDP transport, so it cannot be forwarded using the pre-existing CO
 
 ## Problems
 
-Classic HTTP CONNECT proxies are identified by an origin.  The proxy does not have a path of its own.  This prevents any origin from hosting multiple distinct proxy services.
+Classic HTTP CONNECT proxies are usually identified by an "http" or "https" URI ({{?RFC9110, Section 7.3.2}}).  However, this URI cannot specify a non-empty path, because classic CONNECT requests cannot convey any path information.  This prevents any HTTP origin from hosting multiple distinct proxy services.
```

Comment:
I'm not sure if the second and third sentence are accurate. It's not only the path; none of the proxy's URI component are conveyed by a CONNECT request. A CONNECT proxy is not an origin in terms of HTTP, it is a proxy.

Maybe something like:
```suggestion
HTTP proxies are usually identified by an "http" or "https" URI ({{?RFC9110, Section 7.3.2}}).  However, HTTP requests do not convey any of the proxy's URI components.  Instead, they only carry the URI of the request target (i.e., the origin) or, in the case of CONNECT requests, the host and port of the target.  This prevents any HTTP server from hosting multiple distinct proxy services.
```